jas/applicability/applicabilitymanager.go

```diff
@@ -17,18 +17,22 @@ import (
 )
 
 const (
-	applicabilityScanType      = "analyze-applicability"
-	applicabilityScanCommand   = "ca"
-	applicabilityDocsUrlSuffix = "contextual-analysis"
+	ApplicabilityScannerType        = "analyze-applicability"
+	applicabilityScanCommand        = "ca"
+	applicabilityDocsUrlSuffix      = "contextual-analysis"
+	ApplicabilityDockerScanScanType = "analyze-applicability-docker-scan"
 )
 
+type ApplicabilityScanType string
```

Comment:
```suggestion
	applicabilityScanCommand        = "ca"
	applicabilityDocsUrlSuffix      = "contextual-analysis"

    ApplicabilityScannerType        ApplicabilityScanType = "analyze-applicability"
	ApplicabilityDockerScanScanType ApplicabilityScanType = "analyze-applicability-docker-scan"
)

type ApplicabilityScanType string
```
define the consts as the type